Gemfile

```diff
@@ -11,6 +11,9 @@ gem 'parallel_tests'
 # Debugging
 gem 'pry'                                   # Easily debug from your console with `binding.pry`
 
+# Code style
+gem 'rubocop'
```

Comment:
But this is not your application's `Gemfile`, this is the gem's `Gemfile`. Meaning, it doesn't introduce any runtime dependencies and it's only used for developing the gem. This is a common confusion, I've there too :)